wondrous-app/components/Settings/TaskImport/GenericImportTaskModal/styles.tsx

```diff
@@ -0,0 +1,290 @@
+import {
+  Accordion,
+  AccordionDetails,
+  AccordionSummary,
+  CircularProgress,
+  MenuItem,
+  Modal,
+  Select,
+  Typography,
+} from '@mui/material';
+import styled from 'styled-components';
+import CloseModalIcon from 'components/Icons/closeModal';
+import AttachFileSvg from 'components/Icons/attachFile.svg';
+import { Button } from 'components/Common/button';
+import typography from 'theme/typography';
+import palette from 'theme/palette';
+
+export const GenericImportTaskModalWrapper = styled(Modal)`
+  height: 100vh;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+`;
+
+export const GenericImportTaskModalCard = styled.div`
+  width: 100%;
+  max-width: 560px;
+  background: ${palette.grey900};
+  border: 1px solid ${palette.grey79};
+  border-radius: 6px;
+  overflow: hidden;
+`;
+
+export const GenericImportTaskModalHeader = styled.div`
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  background: ${palette.grey920};
+  padding: 12px 24px;
+`;
+
+export const GenericImportTaskModalHeaderText = styled(Typography)`
+  && {
+    font-family: ${typography.fontFamily};
+    font-weight: 700;
+    font-size: 18px;
+    line-height: 20px;
+    color: ${palette.white};
+  }
+`;
+
+export const GenericImportTaskModalHeaderCloseModalIcon = styled((props) => (
+  <div {...props}>
+    <CloseModalIcon />
+  </div>
+))`
+  width: 32px;
+  height: 32px;
+  background: ${palette.grey100};
+  border-radius: 6px;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  cursor: pointer;
+  svg {
+    transform: scale(88%);
+  }
+  box-shadow: none;
+  transition: box-shadow 0.2s ease-out;
+
+  :hover {
+    box-shadow: 0px 6px 7px ${palette.black}73;
+  }
+`;
+
+export const GenericImportTaskModalBody = styled.div`
+  padding: 24px;
+  background: ${palette.grey900};
+  display: flex;
+  flex-direction: column;
+  gap: 26px;
+`;
+
+export const GenericImportTaskModalInputWrapper = styled.div`
+  display: flex;
+  flex-direction: column;
+  gap: 10px;
+`;
+
+export const GenericImportTaskModalLabel = styled.label`
+  font-family: ${typography.fontFamily};
+  font-weight: 500;
+  font-size: 14px;
+  letter-spacing: 1%;
+  color: ${palette.blue20};
+  width: max-content;
+`;
+
+export const GenericImportTaskModalLabelHelperLink = styled.a`
+  color: ${palette.highlightBlue};
+  margin-left: 6px;
+`;
+
+export const GenericImportTaskModalError = styled(Typography)`
+  && {
+    font-family: ${typography.fontFamily};
+    font-weight: 500;
+    font-size: 12px;
+    line-height: 14px;
+    color: ${palette.red400};
+    padding-left: 4px;
+  }
+`;
+
+const StyledSelect = styled(Select)`
+  && {
+    background: ${palette.background.default};
+    color: ${palette.white};
+    height: 42px;
+    width: 100%;
+    border-radius: 6px;
+    border: 1px solid transparent;
+    transition: border 0.2s ease-out;
+
+    :hover {
```

Comment:
same as above

Reply:
Thank you, didn't know about styled-component using this. Fixed!